lstchain/reco/r0_to_dl1.py

```diff
@@ -566,9 +588,14 @@ def r0_to_dl1(
 
     if is_simu:
         # Reconstruct source position from disp for all events and write the result in the output file
-        for tel_name in ['LST_LSTCam']:
-            focal = OpticsDescription.from_name(tel_name.split('_')[0]).equivalent_focal_length
-            add_disp_to_parameters_table(output_filename, dl1_params_lstcam_key, focal)
+        for telescope_id in subarray.tels.keys():
+            if telescope_id in config['source_config']['EventSource']['allowed_tels']:
```

Comment:
just `source.allowed_tels`?

Reply:
> @mexanick thanks for this PR (see #633 for the initial reported problem).
> Some of the tests are not passing related to the expected naming of some of the tables we use for the analysis. We need to have a careful look and converge to solve this issue, but we are now on the verge of a new release, so I would say that this should go to the first bugfix release of v0.7, is that ok with you?
> (more comments will follow as soon as we have finished with the release)

To my understanding, the tests which aren't passing now should pass (at least part of them) after merging of #641 which will (at least partially) resolve the naming. But indeed, the naming scheme has to be centrally defined. Holding it to past the 0.7 release is ok to me if you want, however I need to proceed with other steps of the simulation asap. This naming story actually makes us troubles as we are working not only on standard LST simulations, but also on SiPM-equipped camera which has different geometry and different name. So instead of hardcoding various things here and there I'd like to resolve the underlying issue.